packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/delete-node/delete-node.routing.js

```diff
@@ -0,0 +1,36 @@
+import { STATUS } from '../../../../../../../components/project/storages/databases/databases.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.databases.dashboard.general-information.delete-node',
+    {
+      url: '/delete-node',
+      views: {
+        modal: {
+          component: 'ovhManagerPciProjectDatabaseGeneralInformationDeleteNode',
+        },
+      },
+      layout: 'modal',
+      resolve: {
+        breadcrumb: () => null,
+        onNodeDelete: /* @ngInject */ (
+          database,
+          getNodeObject,
+          goToDatabase,
+          pollNodesStatus,
+        ) => (nodeInfo, message, type) => {
+          database.setNodeStatus(nodeInfo, STATUS.DELETING);
+          pollNodesStatus();
```

Comment:
not waiting for poll to finish?

Reply:
This function starts the poll on the general-information page, where the nodes status are handled. In the delete-node modale, we remove the node, then go back to the general-information page with the polling activated.

Reply:
What if we leave the general information page, are we killing the polling ?

Reply:
Yes, the general information kills all pollers on destroy.